### PR TITLE
complete javascript-loops

### PR DESCRIPTION
**Pull request by K-Karneboge**
image

### CODE REVIEW

File: javascript-loops/main.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  var keys = [];
+  for (object.keys in object) {
+    keys.push(object.keys);
+  }
+  return keys;
+}
+
+function getValues(object) {
+  var values = [];
+  for (var i = 0; i < object.length; i++) {
```
Comment (BrettAlbright) [inline javascript-loops/main.js]: you want to make sure to use a `for in` loop here. Objects do not have a length property.

File: javascript-loops/main.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+function doubleAll(numbers) {
+  var doubled = [];
+  for (var i = 0; i < numbers.length; i++) {
+    doubled.push(numbers[i] * 2);
+  }
+  return doubled;
+}
+function getKeys(object) {
+  var keys = [];
+  for (object.keys in object) {
```
Comment (BrettAlbright) [inline javascript-loops/main.js]: here you want to define a variable for the for in loop

for (var property in object)

then you have access to the individual keys.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Looking good! A couple small updates. Make sure to reach out if you have questions!

**Comment by K-Karneboge** [comment]
image

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**